core/src/com/unciv/logic/automation/WorkerAutomation.kt

```diff
@@ -367,20 +378,12 @@ class WorkerAutomation(
             !civInfo.isPlayerCivilization() && evaluateFortPlacement(tile, civInfo,false) -> Constants.fort
             // I think we can assume that the unique improvement is better
             uniqueImprovement != null && tile.canBuildImprovement(uniqueImprovement, civInfo)
-                    && unit.canBuildImprovement(uniqueImprovement, tile) ->
-                uniqueImprovement.name
+                -> uniqueImprovement.name
 
             lastTerrain.let {
                 isUnbuildableAndRemovable(it) &&
                         (Automation.rankStatsValue(it, civInfo) < 0 || it.hasUnique(UniqueType.NullifyYields) )
             } -> Constants.remove + lastTerrain.name
-            tile.terrainFeatures.contains(Constants.jungle) -> Constants.tradingPost
```

Comment:
Android Studio says this is the only use of Constants.tradingPost, so can it be removed from Constants.kt?